cmd/ciImageBuild.go

```diff
@@ -56,6 +56,10 @@ var ciImageBuildCmd = &cobra.Command{
 		if _, err := os.Stat(fmt.Sprintf("%s/.dockerignore", buildPath)); err == nil {
 			excludeDockerignore = fmt.Sprintf("--exclude-from='%s'/.dockerignore", buildPath)
 		}
+		// use <dockerfile>.dockerignore if exists, later takes precedence
+		if _, err := os.Stat(fmt.Sprintf("%s.dockerignore", dockerfile)); err == nil {
```

Comment:
I believe you are missing the buildPath here, this now looks for the file in project root.

Reply:
The dockerfile is full path to actual dockerfile and the preferred .dockerignore file location now is `<dockerfile>.dockerfile`. I think this is correct? :thinking:

Reply:
Ah indeed, you are correct!